.size-snapshot.json

```diff
@@ -1,25 +1,25 @@
 {
   "dist/react-beautiful-dnd.js": {
-    "bundled": 393181,
-    "minified": 147695,
-    "gzipped": 41532
+    "bundled": 437376,
+    "minified": 160742,
+    "gzipped": 46554
   },
   "dist/react-beautiful-dnd.min.js": {
-    "bundled": 324803,
-    "minified": 116622,
-    "gzipped": 33468
+    "bundled": 369034,
+    "minified": 129660,
+    "gzipped": 38532
```

Comment:
ouch! this is a 9% change

Reply:
Relevant? 

- https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md
- https://github.com/zloirock/core-js/blob/master/README.md#configurable-level-of-aggressiveness

Reply:
I feel like we are not doing something right to get this size bump

Reply:
https://babeljs.io/blog/2019/03/19/7.4.0#migration-from-core-js-2

I've read through the migration docs for the 7.4.0 release. It basically says to do this:

package.json:

```diff
- @babel/runtime-corejs2
+ @babel/runtime-corejs3
```

babel config:

```diff
plugins: [
  ["@babel/transform-runtime", {
-    corejs: 2,
+    corejs: 3,
  }]
]
```

If anything looks missed, let me know -- but I'm thinking size bump is correct.

Reply:
I wonder if we need to change something. Where is a near 10% increase coming from?